app/services/news-banner.ts

```diff
@@ -0,0 +1,85 @@
+import { StatefulService, mutation } from './stateful-service';
+import { UserService } from './user';
+import { HostsService } from './hosts';
+import { Inject } from '../util/injector';
+import { authorizedHeaders } from '../util/requests';
+
+interface INewsBannerInfo {
+  id: number;
+  header: string;
+  subHeader: string;
+  link: string;
+  linkTitle: string;
+  thumbnail: string;
+}
+
+export class NewsBannerService extends StatefulService<INewsBannerInfo> {
```

Comment:
Maybe `NewsService` or `AnnouncementsService` ? `Banner` is a good name for UI component but our services usually combine wider logic